File: 217-ContainsDuplicate/217-ContainsDuplicate.java
// Last updated: 6/27/2025, 2:30:13 AM
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> number = new HashSet<>();
        for(int index = 0; index < nums.length; index++){
            if(number.contains(nums[index])){
                return true;
            }
            else{
                number.add(nums[index]);
            }
        }
        return false;
    }
}

File: 217-ContainsDuplicate/217-ContainsDuplicate.cpp
// Last updated: 6/27/2025, 2:30:06 AM
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        set<int> s;
        for(int i = 0; i < nums.size(); i++){
            if(s.find(nums[i]) != s.end()){
                return true;
            } else{
                s.emplace(nums[i]);
            }
        }
        return false;
    }
};